README.rst

```diff
@@ -266,6 +266,32 @@ command:
 
 This will open up the jupyter hub and will allow you to work on your notebooks.
 
+It is also possible to develop with both GeoPySpark and GeoNotebook in editable mode.
+To do so you will need to re-install and re-register GeoNotebook with Jupyter.
+
+.. code:: console
+
+   pip uninstall geonotebook
+   git clone --branch feature/geotrellis https://github.com/geotrellis/geonotebook ~/geonotebook
+   pip install -e ~/geonotebook
+   jupyter serverextension enable --py geonotebook
+   jupyter nbextension enable --py geonotebook
+   make notebook
+
+The default `Geonotebook (Python 3)` kernel will require the following environment variables to be defined:
```

Comment:
Having a single ` in a rst file makes the word/s italicized. If you wanted to do an in line quote, you need to use ``